contrib/signet/getcoins.py

```diff
@@ -120,22 +122,34 @@ def bitcoin_cli(rpc_command_and_params):
 if args.captcha != '': # Retrieve a captcha
     try:
         res = session.get(args.captcha)
-    except:
-        print('Unexpected error when contacting faucet:', sys.exc_info()[0])
-        exit(1)
+        res.raise_for_status()
+    except requests.exceptions.RequestException as e:
+        raise SystemExit(f"Unexpected error when contacting faucet: {e}")
+
+    # Size limitation
+    svg = xml.etree.ElementTree.fromstring(res.content)
+    if (svg.attrib.get('width') != '150' or svg.attrib.get('height') != '50'):
+        raise SystemExit("Captcha dimentions doesn't satisify")
```

Comment:
This is not correct English, "Captcha size doesn't match expected 150x50" maybe

Reply:
Okay, my bad!